### PR TITLE
merge

### PR DESCRIPTION
**Pull request by jonborchardt**
@codeviking please review changes to package.json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,11 +54,12 @@
     "presite": "npm run version",
     "color-less": "node ./scripts/generate-color-less",
     "compile": "npm run shellac:build && antd-tools run compile",
+    "compile:less": "antd-tools run compile:less",
```
Comment (codeviking) [inline package.json:57]: Do we need to run this now? I don't see it wired into any of the other steps?

Comment (jonborchardt) [reply package.json:57]: its used by test:image
btw, they are using docker!!

Comment (jonborchardt) [reply package.json:57]: i dont see how they run this int he upstream, so i think we are ok

File: package.json [hunk 2]
```diff
@@ -81,11 +83,12 @@
     "test:update2": "npm test -- -u",
     "test-all": "./scripts/test-all.sh",
     "test-node": "jest --config .jest.node.js --no-cache",
-    "tsc": "tsc",
+    "tsc": "tsc --noEmit",
```
Comment (codeviking) [inline package.json:86]: ☝️

**Pull request merged**